localstack/services/stepfunctions/asl/component/state/state_execution/state_task/service/state_task_service_sns.py

```diff
@@ -84,6 +85,13 @@ def _eval_service_task(
             account=resource_runtime_part.account,
             service=service_name,
         )
+
+        # Optimised integration automatically stringifies
+        if "Message" in normalised_parameters:
+            message = normalised_parameters["Message"]
+            if not isinstance(message, str):
+                normalised_parameters["Message"] = to_json_str(message)
```

Comment:
```suggestion
        if message := normalized_parameters.get("Message"):
            if not isinstance(message, str):
                normalized_parameters["Message"] = to_json_str(message)
```
bit of a nit (walrus operator)

also we generally try to use American English for consistency

Reply:
Thank you, Dominik! The walrus operator really improves readability in this case. I changed it slightly to ensure it wouldn't add missing 'Message' parameters (in case this is not given by the program).

About the British spelling, I realise it's a problem for this provider, I'll open a PR to standardise all of it!